client/my-sites/stats/stats-download-csv/index.jsx

```diff
@@ -55,8 +65,32 @@ module.exports = React.createClass( {
 
 		return (
 			<Button compact onClick={ this.downloadCsv }>
```

Comment:
Yeah I thought the same thing.  It has never been an issue, but seems like a good thing to have.